mentat/conversation.py

```diff
@@ -153,11 +156,21 @@ async def get_model_response(
         self, parser: Parser, config: ConfigManager
     ) -> list[FileEdit]:
         messages = self.messages.copy()
+
+        # Rebuild code context with active code and available tokens
+        conversation_history = "\n".join([m["content"] for m in messages])
```

Comment:
Probably doesn't really matter but it looks like this isn't quite [right](https://github.com/openai/openai-cookbook/blob/main/examples/How_to_count_tokens_with_tiktoken.ipynb). (ctrl-f "every reply is primed with")

Maybe we should overload our own `count_tokens` method to accept an object that looks like a llm conversation.

(may be better to do this as a smaller follow pr)

Reply:
The openai-cookbook is under an MIT License so we can just use that function, right? I'll add that.